cumulus_library/studies/core/core_templates/medicationrequest.sql.jinja

```diff
@@ -64,60 +64,69 @@ CREATE TABLE core__medicationrequest AS (
+        SELECT
+            mr.id,
+            mrcc.code AS medication_code,
+            mrcc.system AS medication_system,
+            mrcc.display AS medication_display
+        FROM mr_basics AS mr
+        INNER JOIN contained_refs AS cr ON mr.id = cr.id
+        INNER JOIN core__medicationrequest_dn_contained_code AS mrcc
+            ON cr.id = mrcc.id AND cr.medication_id = mrcc.contained_id
+        WHERE mrcc.resource_type = 'Medication'
+
+        {# External: medication by reference from external medications table.
+        This is generally how we expect EPIC to provide medication data. #}
+        UNION
+        SELECT
+            mr.id,
+            mc.code AS medication_code,
+            mc.system AS medication_system,
+            mc.display AS medication_display
+        FROM mr_basics AS mr
+        INNER JOIN external_refs AS er ON mr.id = er.id
+        INNER JOIN core__medication_dn_code AS mc ON er.medication_id = mc.id
     )
 
-    {# Internal: medication data from inline ETL extraction.
-    95% of the time, this dataset is the 'correct' dataset for Cerner.
-    It may not be present in EPIC datasets. #}
     SELECT
-        mr.*,
-        mric.code AS medication_code,
-        mric.system AS medication_system,
-        mric.display AS medication_display
-    FROM mr_shared AS mr
-    INNER JOIN core__medicationrequest_dn_inline_code AS mric ON mr.id = mric.id
+        mr.id,
+        mr.status,
+        mr.intent,
 
-    {# Contained: medication reference into contained resources.
-    We've also seen this in Cerner. Though in the cases we've seen this, it's usually
-    a very lightly fleshed out resource, with only code.text and no codings. #}
-    UNION
-    SELECT
-        mr.*,
-        mrcc.code AS medication_code,
-        mrcc.system AS medication_system,
-        mrcc.display AS medication_display
-    FROM mr_shared AS mr
-    INNER JOIN contained_refs AS cr ON mr.id = cr.id
-    INNER JOIN core__medicationrequest_dn_contained_code AS mrcc
-        ON cr.id = mrcc.id AND cr.medication_id = mrcc.contained_id
-    WHERE mrcc.resource_type = 'Medication'
+        mrc.code AS category_code,
+        mrc.system AS category_system,
+        mrc.display AS category_display,
 
-    {# External: medication by reference from external medications table.
-    This is generally how we expect EPIC to provide medication data. #}
-    UNION
-    SELECT
-        mr.*,
-        mc.code AS medication_code,
-        mc.system AS medication_system,
-        mc.display AS medication_display
-    FROM mr_shared AS mr
-    INNER JOIN external_refs AS er ON mr.id = er.id
-    INNER JOIN core__medication_dn_code AS mc ON er.medication_id = mc.id
+        mr.reportedBoolean,
+        mr.reported_ref,
+
+        uc.medication_code,
+        uc.medication_system,
+        uc.medication_display,
+
+        mr.authoredOn,
+        mr.authoredOn_month,
+
+        concat('MedicationRequest/', mr.id) AS medicationrequest_ref,
```

Comment:
It's a little hard to see because I rearranged how this table gets generated (to more easily inject the code triplet in the middle of the list). But this line here is new - I added it because it seems standard for core tables to have this kind of field (and it makes sense).